package.json

```diff
@@ -1,61 +1,62 @@
 {
-	"name": "array.from",
-	"version": "1.0.1",
-	"description": "A robust & optimized `Array.from` polyfill, based on the ECMAScript 6 specification.",
-	"homepage": "https://mths.be/array-from",
-	"main": "index.js",
-	"keywords": [
-		"array",
-		"es6",
-		"ecmascript",
-		"polyfill"
-	],
-	"license": "MIT",
-	"author": {
-		"name": "Mathias Bynens",
-		"url": "https://mathiasbynens.be/"
-	},
-	"contributors": [
-		{
-			"name": "Mathias Bynens",
-			"url": "https://mathiasbynens.be/"
-		},
-		{
-			"name": "Jordan Harband",
-			"email": "ljharb@gmail.com",
-			"url": "http://ljharb.codes"
-		}
-	],
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/mathiasbynens/Array.from.git"
-	},
-	"bugs": "https://github.com/mathiasbynens/Array.from/issues",
-	"files": [
-		"LICENSE-MIT.txt",
-		"implementation.js",
-		"index.js",
-		"polyfill.js",
-		"shim.js"
-	],
-	"directories": {
-		"test": "tests"
-	},
-	"scripts": {
-		"test": "npm run lint && es-shim-api --bound && npm run tests-only",
-		"tests-only": "node tests/*.js",
-		"cover": "covert tests/*.js",
-		"lint": "eslint *.js tests/*.js"
-	},
-	"devDependencies": {
-		"@es-shims/api": "^1.0.0",
-		"covert": "^1.1.0",
-		"tape": "^4.4.0",
-		"eslint": "^1.10.3",
-		"@ljharb/eslint-config": "^1.6.1"
-	},
-	"dependencies": {
-		"define-properties": "^1.1.2",
-		"es-abstract": "^1.5.0"
-	}
+  "name": "array.from",
```

Comment:
please revert all the spacing changes in package.json :-/ it's always best to manually edit it instead of letting npm do it.